### PR TITLE
UPDATE #455: Add instructions for build tools to CONTRIBUTING.md

### LINKED ISSUE #455: Add instructions for build tools to CONTRIBUTING.md
As we've introduced build tools recently, the steps to run them should be mentioned in CONTRIBUTING.md. 

I'm working on this issue and will create a PR within the next minutes.

### PR DESCRIPTION
**Pull request by nielslange**
Fixes #455 

Added the following section:

## Create Right-To-Left (RTL) CSS file

Twenty Twenty uses [RTLCSS](https://rtlcss.com/) which allows to transform the  Left-To-Right (LTR) file `style.css` into the Right-To-Left (RTL) file `style-rtl.css`.

### Installation instructions

1. Using a command line interface, go to the `twentytwenty` directory `cd /my-compter/local-wordpress-install/wp-content/themes/twentytwenty`.
2. Type `npm install` into the command line and press [return], to install all Node.js dependencies. 
3. The dependencies may take a few minutes to download and once it completes, you’re done.

### Usage instructions

1. After making a change to the `style.css` file, which affects RTL, run `npm run build:rtl` from within the theme directory to build the `style-rtl.css` file with your new changes.

### CODE REVIEW

**Comment by joyously** [comment]
@nielslange 
There was an open question in one of the RTL issues about whether there are RTL versions of the editor stylesheets.

**Comment by nielslange** [comment]
@joyously I must have missed that and, in my opinion, the editor stylsheets should also be converted to ensure that they show the correct direction. I'll analyse this issue and enhance the PR if needed.

File: package.json [hunk 1]
```diff
@@ -30,6 +30,8 @@
   },
   "scripts": {
     "build:rtl": "rtlcss style.css style-rtl.css",
+    "build:rtl-esb": "rtlcss assets/css/editor-style-block.css assets/css/editor-style-block-rtl.css",
```
Comment (nielslange) [inline package.json]: @ntwb Would this be the correct way to create the RTL file for `assets/css/editor-style-block.css`?

File: package.json [hunk 2]
```diff
@@ -30,6 +30,8 @@
...
+    "build:rtl-esc": "rtlcss assets/css/editor-style-classic.css assets/css/editor-style-classic-rtl.css",
```
Comment (nielslange) [inline package.json]: @ntwb Would this be the correct way to create the RTL file for `assets/css/editor-style-classic.css`?

**Comment by nielslange** [comment]
@joyously As you can tell, I've enhanced the PR for this issue as the editor stylesheet files contained a few align statements. Thanks for pointing out this issue.

**Comment by joyously** [comment]
@nielslange  Does that include getting the new ones loaded?

**Comment by nielslange** [comment]
> @nielslange Does that include getting the new ones loaded?
That would be good good to be true, wouldn't it! I totally missed enqueueing them. 🤦‍♂️

**Comment by joyously** [comment]
Well, the add_editor_stylesheet function does allow for RTL. I just wanted to make sure that's what was happening.

**Comment by nielslange** [comment]
@joyously I simply forgot to enqueue the RTL editor stylesheets initially. Also, if I understand https://developer.wordpress.org/reference/functions/add_editor_style/ correctly, then I don't need to enqueue `/assets/css/editor-style-classic-rtl.css` but only `/assets/css/editor-style-block-rtl.css`. Can you confirm that?

**Comment by joyously** [comment]
The function only adds the RTL for the first stylesheet passed. If you are loading the block editor sheet differently, then you will have to do your own load for RTL.

**Comment by nielslange** [comment]
In this case, it should work already. But I’ll keep it in the mind when coordinating the testing for https://github.com/WordPress/twentytwenty/issues/118.

File: package.json (hunk 2 above)
Comment (ntwb) [reply package.json]: Close, I’ll add some functionality so that there is only a single bpm script to run instead of 3

**Comment by ntwb** [comment]
I've update the scripts so that the single npm script `npm run build:rtl` builds all 3 RTl _concurrently_ (in parallel as a single task)

**Comment by nielslange** [comment]
> I've update the scripts so that the single npm script `npm run build:rtl` builds all 3 RTl _concurrently_ (in parallel as a single task)

Thanks, @ntwb! Your solution is much more elegant. I'll keep that solution in mind for related tasks in the future. 😀

**Pull request merged**